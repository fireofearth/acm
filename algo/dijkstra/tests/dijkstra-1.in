5 6
1
1 2 30
1 3 20
1 4 10
2 5 10
3 5 30
4 5 100
