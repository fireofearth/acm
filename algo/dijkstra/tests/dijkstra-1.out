1, cost: 0, path: 1
2, cost: 30, path: 1 2
3, cost: 20, path: 1 3
4, cost: 10, path: 1 4
5, cost: 40, path: 1 2 5
